examples/cisco/demo_vxlan.pp

```diff
@@ -25,35 +25,72 @@
     dup_host_mac_detection_timeout        => '10',
   }
 
```

Comment:
Since this isn't supported on 3k/XR you should have all of this code in an if block starting at line 20 and a notify/skip at the end of the file; e.g.:

```
  if platform_get() =~ /n(5|6|7|8|9)k/ {
    cisco_overlay_global { 'default':

       ...blah...

       ...blah...

  } else {
    notify{'SKIP: This platform does not support vxlan': }
  }

```